ql/src/test/results/clientpositive/llap/avrotblsjoin.q.out

```diff
@@ -73,6 +73,7 @@ POSTHOOK: Lineage: table1_1.col1 SCRIPT []
 POSTHOOK: Lineage: table1_1.col2 SCRIPT []
 WARNING: Comparing a bigint and a string may result in a loss of precision.
 WARNING: Comparing a bigint and a string may result in a loss of precision.
+WARNING: Comparing a bigint and a string may result in a loss of precision.
```

Comment:
Is this expected? Same question for the same WARNING messages in q files below (I assume root cause is the same).

Reply:
It is a side effect that unparsing is enable for all queries.
When the calcite plan is generated `genJoinRelNode` is called and it calls `genAllExprNodeDesc` because unparsing is enabled which is required for expanded query text.
```
        } else if (unparseTranslator != null && unparseTranslator.isEnabled()) {
          genAllExprNodeDesc(joinCond, input, jCtx);
        }
```